initializer_inputs/industry/industry_aluminium_production.ad

```diff
@@ -0,0 +1,3 @@
+- query = UPDATE_WITH_FACTOR(V(industry_aluminium_production), preset_demand, USER_INPUT())
+- priority = 10
+- update_type = %
```

Comment:
The % `update_type` needs to stay because it's used to 'scale' certain industry sectors according to @ChaelKruip